frontend/src/scenes/saved-insights/savedInsightsLogic.ts

```diff
@@ -50,6 +120,45 @@ export const savedInsightsLogic = kea<savedInsightsLogicType<InsightsResult>>({
             },
         ],
     },
+    listeners: ({ actions }) => ({
+        setTab: () => {
+            actions.loadInsights()
+        },
+        setSearchTerm: ({ term }) => {
+            if (term.length === 0) {
+                actions.loadInsights()
+            }
+        },
+        setInsightType: () => {
+            actions.loadInsights()
+        },
+        setCreatedBy: () => {
+            actions.loadInsights()
+        },
```

Comment:
Kind of torn if it's worth the refactor to put all these parameters (tab, search term, dates, type, etc.) under a source of truth `filters` like we have in our insight logics. It would make for less boilerplate code and could persist these options in the url (which is especially helpful if the user wants to navigate back to their saved_insights after clicking an insight row). @mariusandra what are your thoughts?

Reply:
It would indeed be great to persist these variables in the URL... and that's indeed easier to achieve if you have one `filters` reducer, versus multiple other reducers, though this can work as well.